RecoParticleFlow/PFTracking/src/PFDisplacedVertexCandidateFinder.cc

```diff
@@ -18,10 +18,9 @@ PFDisplacedVertexCandidateFinder::PFDisplacedVertexCandidateFinder()
       primaryVertexCut2_(0.0),
       dcaPInnerHitCut2_(1000.0),
       vertexCandidatesSize_(50),
-      debug_(false) {
-  TwoTrackMinimumDistance theMinimum(TwoTrackMinimumDistance::SlowMode);
```

Comment:
Out of curiosity @Dr15Jones, and for my own education: what was uninitialized here?

Reply:
The original error report was
```
TrackingTools/PatternTools/interface/TwoTrackMinimumDistance.h:19:7: runtime error: load of value 32508, which is not a valid value for type 'Charge'
```